capstone/src/main/java/com/google/univiz/UniversityData.java

```diff
@@ -0,0 +1,105 @@
+package com.google.univiz;
+
+public final class UniversityData {
```

Comment:
Consider leveraging [AutoValue](https://github.com/google/auto/blob/master/value/userguide/index.md) for this class. This has a number of advantages and will prevent pitfalls (like not overriding hashcode, equals, etc). See also Effective Java chapter 3.

I'm adding a dependency on AutoValue in https://github.com/googleinterns/univiz/pull/33 - so feel free to rebase off of that branch or wait until I merge...

Reply:
Did this get addressed? I think it will improve this class.